### PR TITLE
Improve error message when server can not connect to node

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java [hunk 1]
```diff
@@ -1178,7 +1177,8 @@ private RMAuthentication joinResourceManager(String rmURL) {
             logger.info("Resource Manager joined.");
             return auth;
         } catch (Throwable t) {
-            logger.error("Unable to join the Resource Manager at " + rmURL, t);
+            logger.error("Unable to join the Resource Manager at " + rmURL
+            + "\n ProActive server can not send reply to Node (usually due to firewall). Please check server log for more information", t);
```
Comment (mboussaa) [inline rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java]: ...can not request the Node...?

Comment (aminelouati) [reply rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java]: cannot connect to the Node

Comment (fviale) [reply rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java]: it's not here that the error message should be changed

**Comment by ShatalovYaroslav** [comment]
retest it please

**Comment by ShatalovYaroslav** [comment]
test this please

**Comment by MaelAudren** [comment]
@ShatalovYaroslav can you squash your commit before merging ?

**Pull request merged**